src/space-connector/type.ts

```diff
@@ -2,11 +2,41 @@ export interface SessionTimeoutCallback {
     (): void;
 }
 
+export type ResponseCode = 'APIError' | 'NotFoundError' | 'BadRequestError' | 'AuthenticationError' | 'AuthorizationError'
+
+export interface Response<T> {
+    code: ResponseCode | string;
+    message: string;
+    result: T;
+}
+
 export interface APIInfo {
     path: string;
     methods: Array<string>;
 }
 
+export interface ErrorModel {
+    message: string;
+    code: string;
+}
+
+export interface ServerError {
+    status: string;
+    error: ErrorModel;
+    statusText: string;
+}
+
+export interface AxiosPostResponse {
+    access_token: string;
+    refresh_token: string;
+    apis: APIInfo[];
+}
```

Comment:
`AxiosPostResponse` 가 쓰이는 용례를 보니, 모두 `AxiosResponse<AxiosPostResponse>` 형태로 쓰이는 것을 확인했습니다.
그러면 사용하는 쪽에서는 매번 AxiosResponse와 AxiosPostResponse를 알아야 하고, 모두를 import 해야 하니
그 수고를 줄이기 위해서 아래처럼 제공해주는 것은 어떨까요?

```suggestion
interface PostResponse {
    access_token: string;
    refresh_token: string;
    apis: APIInfo[];
}

export type AxiosPostResponse = AxiosResponse<PostResponse>;
```